README.md

```diff
@@ -143,6 +143,20 @@ By default, jQuery generates a regular script JavaScript file. You can also gene
 npm run build -- --filename=jquery.module.js --esm
 ```
 
+##### Factory mode
+
+By default, jQuery depends on a global `window`. For environments that don't have one, you can generate a factory build that exposes a function accepting `window` as a parameter that you can provide externally (see [`README` of the published package](build/fixtures/README.md) for usage instructions). You can generate such a factory using the `--factory` parameter:
+
+```bash
+grunt custom --factory
+```
+
+The default mode doesn't generate a factory, but you can also pass it explicitly via `--no-factory`:
+
+```bash
+grunt custom --no-factory
+```
+
```

Comment:
Need to update the build/command.js and add the boolean argument `factory`. The equivalent of `--no-factory` would be `--factory=false`. Then update docs here.

Reply:
I think I'll skip the `false` versions since we don't cover it for other options either.

Reply:
Updated.